src/main/java/io/github/nucleuspowered/nucleus/modules/back/listeners/OfflineLocation.java

```diff
@@ -32,6 +33,15 @@ public OfflineLocation(Transform<World> location) {
         if (this.lastWorld == null || this.lastRotation == null || this.lastPosition == null) {
             return Optional.empty();
         }
+        
+        Optional<WorldProperties> optionalWorldProperties = Sponge.getServer()
+                .getUnloadedWorlds().stream().filter(worldProps -> worldProps.getUniqueId().equals(this.lastWorld))
+                .findFirst();
+
+        optionalWorldProperties.ifPresent(worldProperties -> {
+            Optional<World> worldOptional = Sponge.getServer().getWorld(worldProperties.getUniqueId());
+            if (!worldOptional.isPresent()) Sponge.getServer().loadWorld(worldProperties);
+        });
 
         return Sponge.getServer().getWorld(this.lastWorld)
```

Comment:
Can't you just do this instead?

```suggestion
        return Sponge.getServer().loadWorld(this.lastWorld)
```